nrt/data/DynaFlow/VoltageMeasurementsUtilities/solvers.par

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2021, RTE (http://www.rte-france.com)
+    See AUTHORS.txt
+    All rights reserved.
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, you can obtain one at http://mozilla.org/MPL/2.0/.
+    SPDX-License-Identifier: MPL-2.0
+
+   This file is part of Dynawo, an hybrid C++/Modelica open source suite of
+   simulation tools for power systems.
+-->
+<parametersSet xmlns="http://www.rte-france.com/dynawo">
```

Comment:
The nrt should be added on the private repo with some references. I'll let you check with Marco or Quentin on how to properly create it

Reply:
This will be done a posteriori, by someone with right access to the private repo

Reply:
As you have the proper access to the private repo, this could be done just when this is validated